cloudbeesjenkins-dockerdatacenter/README.md

```diff
@@ -7,34 +7,37 @@
+***Azure Partner Quickstart Templates*** provide customers with a highly automated process to launch enterprise ready 1st and 3rd party ISV solution stacks on Azure in a pre-production environment. The **Azure Partner Quickstart** effort is complimentary to the [Azure Marketplace test drive program](https://azure.microsoft.com/en-us/blog/azure-marketplace-test-drive/). These fully baked stacks enable customers to quickly stand up a Proof of Concept or Pilot environments, as well as integrate it with their existing systems and configurations.
 
-Customers benefit greatly from solution templates because of the ease with which they can stand up enterprise-grade, fully integrated stacks on Azure. The extensive automation and testing of these solutions will allow them to spin up pre-production environments with minimal manual steps and customization.  Most importantly, customers now have the confidence to transition the solution into a fully production-ready environment with confidence.
+Customers benefit greatly from solution templates because of the ease with which they can stand up enterprise-grade, fully integrated stacks on Azure. The extensive automation and testing of these solutions will allow them to spin up pre-production environments with minimal manual steps and customization.  Most importantly, customers can have the confidence to transition the solution into a fully production-ready environment once ready.
 
-**The Continuous Integration & Deployment Azure Partner Quickstart Template** launches a CI/CD solution stack that provides an automated provisioning, configuration and integration of the [Docker Datacenter](https://azure.microsoft.com/en-us/marketplace/partners/docker/dockerdatacenterdocker-datacenter/) product on Azure. Combined with [Cloudbees Jenkins platform](https://azure.microsoft.com/en-us/marketplace/partners/cloudbees/jenkins-platformjenkins-platform/) products makes this solution ready for pre-production environments. These are intended as pilot solutions and are not production-ready.
-Please [contact us](azuremarketplace@sysgain.com) if you need further info or support on this solution.
+**The Continuous Integration & Deployment (CI/CD) Azure Partner Quickstart Template** launches a CI/CD solution stack that provides  automated provisioning, configuration and integration of the [Docker Enterprise Edition](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/docker.dockerdatacenter) product on Azure, combined with the [Cloudbees Jenkins platform](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/cloudbees.jenkins-platform) product. This stack allows customer to deploy a CI/CD workflow and start building, shipping, and deploying containerized apps in a pre-production environments. These are intended as pilot solutions and are not production-ready.
```

Comment:
"provides  automated provisioning" - typo has an extra space...

"pre-production environments." - environments should be singular